cli/smartcontract/contract_test.go

```diff
@@ -358,13 +360,20 @@ func TestContractDeployWithData(t *testing.T) {
 		}
 		e.In.WriteString(testcli.ValidatorPass + "\r")
 		e.Run(t, cmd...)
+		var tx *transaction.Transaction
+		if await {
+			tx, _ = e.CheckTxPersisted(t)
+			e.CheckNextLine(t, "OnChain:\ttrue")
+			e.CheckNextLine(t, "BlockHash:\t"+tx.Hash().StringLE())
+			e.CheckNextLine(t, "VMState:\tHALT")
+		} else {
```

Comment:
Please, add the following helper under CheckTxPersisted and call it where appropriate in tests:
```


func (e *Executor) CheckAwaitableTxPersisted(t *testing.T, prefix ...string) (*transaction.Transaction, uint32) {
	tx, vub := e.CheckTxPersisted(t, prefix...)
	e.CheckNextLine(t, "OnChain:\ttrue")
	e.CheckNextLine(t, "VMState:\tHALT")
	return tx, vub
}
```